src/components/metadata/styles.js

```diff
@@ -0,0 +1,48 @@
+const styles = (theme) => ({
+    metadataTableContainer: {
+        overflow: "auto",
+    },
+    attributeTableContainer: {
+        width: "100%",
+        height: "100%",
+    },
+    flex: {
+        flex: 1,
+    },
+    tableHead: {
+        backgroundColor: theme.palette.lightGray,
+        position: "sticky",
+        top: 0,
+    },
+    deleteIcon: {
+        margin: 5,
+        "&:hover": {
+            backgroundColor: theme.palette.error.dark,
```

Comment:
I tried this out locally and I'm wondering if the dark error color is actually a bit too dark on hover?  It was a little difficult to make out the trash icon with the red. Might just be me though.

![image](https://user-images.githubusercontent.com/8909156/101421223-1a819980-38b1-11eb-8d5c-a84f38a84b36.png)

Reply:
Oops. That style is just a direct import from the current DE, where it's much lighter for some reason. I'll use `error.main` for Sonora instead.